cmd/talisman/talisman.go

```diff
@@ -63,7 +69,10 @@ func doUpgrade() {
 
 func init() {
 	flag.StringVar(&op, "operation", "upgrade", "Operation to perform for the Portworx cluster")
```

Comment:
I'm not sure if you have one already, but if not, add a help screen and examples.

Reply:
the flag packages gives that for free

```
bin/talisman --help
INFO[0000] Running talisman: 70cd9f4-px-upgrade
Usage of bin/talisman:
  -alsologtostderr
    	log to standard error as well as files
  -dockerregsecret string
    	(optional) Kubernetes Secret to pull docker images from a private registry
  -kubeconfig string
    	(optional) Absolute path of the kubeconfig file
  -log_backtrace_at value
    	when logging hits line file:N, emit a stack trace
  -log_dir string
    	If non-empty, write log files in this directory
  -logtostderr
    	log to standard error instead of files
  -ocimonimage string
    	(optional) New OCI Monitor Image to use for the upgrade (default "portworx/oci-monitor")
  -ocimontag string
    	New OCI Monitor tag to use for the upgrade
  -operation string
    	Operation to perform for the Portworx cluster (default "upgrade")
  -pximage string
    	(optional) New Portworx Image to use for the upgrade
  -pxtag string
    	(optional) New Portworx tag to use for the upgrade
  -stderrthreshold value
    	logs at or above this threshold go to stderr
  -v value
    	log level for V logs
  -vmodule value
    	comma-separated list of pattern=N settings for file-filtered logging
```

Reply:
Yeah, but there are no examples. This is the help of each argument. For example, how do I use it?

`talisman -kubeconfig /path/to/config -operation upgrade` , etc.

Reply:
Created https://github.com/portworx/talisman/issues/20 to add that later